creation/web_base/condor_startup.sh

```diff
@@ -760,26 +765,31 @@ EOF
             echo "NUM_SLOTS_TYPE_1 = 1" >> "$CONDOR_CONFIG"
             echo "SLOT_TYPE_1_PARTITIONABLE = True" >> "$CONDOR_CONFIG"
             num_slots_for_shutdown_expr=1
-            #Blackhole calculation based on the parent startd stats (child slots will have the same stats).
-            cat >> "$CONDOR_CONFIG" <<EOF
+            # Blackhole calculation based on the parent startd stats (child slots will have the same stats).
+	    if $use_blackhole_prevention; then
+                cat >> "$CONDOR_CONFIG" <<EOF
 BLACKHOLE_TRIGGERED_P = (RecentJobBusyTimeAvg < (1.0/$glidein_blackhole_rate)) && (RecentJobBusyTimeCount >= $glidein_blackhole_numjobs)
 GLIDEIN_BLACKHOLE = ifThenElse(isUndefined(\$(BLACKHOLE_TRIGGERED_P)), False, \$(BLACKHOLE_TRIGGERED_P))
 EOF
+            fi
```

Comment:
Is it okay if `fi` is indented more than it's opening `if`? I know this is in a shell script and indentation does not matter. Asking this for merely consistency purposes.

Reply:
fixed tabs